src/routes/index.jsx

```diff
@@ -5,13 +5,15 @@ import { Switch, Route } from 'react-router-dom';
 import { Login } from '../screens/Login';
 import { ForgotPassword } from '../screens/ForgotPassword'
 import { Home } from '../screens/Home'
+import { Question } from '../screens/SurveyDetails'
 
 const RouteList = () => {
   return (
     <Switch>
       <Route exact path="/" component={ Login }></Route>
       <Route path="/forgot-password" component={ ForgotPassword }></Route>
       <Route path="/home" component={ Home }></Route>
+      <Route path="/question" component={ Question }></Route>
```

Comment:
SHouldn't the route be like `/survey/:id/question/:id` ?

Reply:
It's for initial loading purpose, we will change it while doing select survey.